api/DeviceMotionEventAcceleration.json

```diff
@@ -23,10 +23,14 @@
             "version_removed": "79"
           },
           "firefox": {
-            "version_added": null
+            "version_added": "6",
+            "partial_implementation": true,
+            "notes": "Doesn't expose the <code>DeviceMotionEventAcceleration</code> interface itself."
```

Comment:
This wording is different from some other similar cases. Can you search for "expose" and "interface" in notes to see if you can find the most common wording to copy?

Reply:
I was copying the notes from Safari, which seems to have the same behavior.  It doesn't seem like there's many of those notes around...

Reply:
OK, let's go with this, I'm sure I'll run into the notes I'm thinking of at some point and make them consistent if needed.